transactions.py with nightly_protocol

`pytest sanity/transactions.py --features nightly_protocol --features nightly_protocol_features`

http://nayduck.eastus.cloudapp.azure.com:3000/#/test/117583
```
Traceback (most recent call last):
  File "tests/sanity/transactions.py", line 81, in <module>
    sent_height, height)
AssertionError: Balances before: [999999999999999999999999999999773, 999999999999999999999999999999786, 1000000000000000000000000000000876, 999999999999999999999999999999429, 1000000000000000000000000000000136]
Expected balances: [999999999999999999999999999999245, 999999999999999999999999999999057, 1000000000000000000000000000001438, 999999999999999999999999999999813, 1000000000000000000000000000000447]
Current balances: [1000000000000000000000000000000187, 999999999999999999999999999999786, 1000000000000000000000000000000876, 999999999999999999999999999999439, 999999999999999999999999999999712]
Sent at height: 20, cur height: 31
```

--- a/pytest/tests/sanity/transactions.py
+++ b/pytest/tests/sanity/transactions.py
@@ -25,7 +25,14 @@ nodes = start_cluster(
         ["epoch_length", 10],
         ["block_producer_kickout_threshold", 70]
     ],
-    client_config_changes={ 4: {"tracked_shards": [0, 1, 2, 3]}}
+    client_config_changes={
+                                  0: {"consensus": {"state_sync_timeout": {"secs": 2, "nanos": 0}}},
+                                  1: {"consensus": {"state_sync_timeout": {"secs": 2, "nanos": 0}}},
+                                  2: {"consensus": {"state_sync_timeout": {"secs": 2, "nanos": 0}}},
+                                  3: {"consensus": {"state_sync_timeout": {"secs": 2, "nanos": 0}}},
+                                  4: {"consensus": {"state_sync_timeout": {"secs": 2, "nanos": 0}},
+                                      "tracked_shards": [0, 1, 2, 3]}
+    }
 )
 
 started = time.time()
@@ -41,7 +48,7 @@ sent_height = -1
 
 while True:
     assert time.time() - started < TIMEOUT
-    status = nodes[3].get_status()
+    status = nodes[4].get_status(check_storage=False)
 
     height = status['sync_info']['latest_block_height']
     hash_ = status['sync_info']['latest_block_hash']

